src/datetimepicker.ios.js

```diff
@@ -109,19 +116,16 @@ export default function Picker({
     return null;
   }
 
-  const dates: DatePickerOptions = {value, maximumDate, minimumDate};
-  toMilliseconds(dates, 'value', 'minimumDate', 'maximumDate');
```

Comment:
it's difficult to cast this since we are changing the value in place.
A better alternative is to have a `dateToMilliseconds` helper fn that returns a `?number`